nginx/docker-entrypoint.d/30-init-kobo-nginx.sh

```diff
@@ -10,26 +10,6 @@ KOBOCAT_PRODUCTION_LOCATION_STATIC='location /static {
 
 KPI_PRODUCTION_LOCATION_STATIC='location /static {
         alias /srv/www/kpi;
```

Comment:
`KPI_PRODUCTION_LOCATION_STATIC` and `KOBOCAT_PRODUCTION_LOCATION_STATIC` are useless now. 
You could remove both of them and replace `${kobocat_location_static}` and `${kpi_location_static}` in `nginx_site_default.conf.tmpl` with the corresponding `location block`.